community/kernel/src/main/java/org/neo4j/kernel/impl/cache/MeasureDoNothing.java

```diff
@@ -23,20 +23,69 @@
 
 public class MeasureDoNothing extends Thread
 {
+    public interface Monitor
+    {
+        default void started()
+        {
+        }
+
+        void blocked( long millis );
+
+        default void stopped()
+        {
+        }
+    }
+
+    public static Monitor loggingMonitor( Log log )
+    {
+        return new Monitor()
+        {
+            @Override
+            public void started()
+            {
+                log.debug( "GC Monitor started. " );
```

Comment:
while this change improves existing code we should stop at some point calling this thing a GC monitor, because its not

Reply:
It's something of a rough indicator of amount of time spent in Full GC

Reply:
pauses of those threads can have nothing to do with GC in general and with full GC in particular but lets keep it away from current pr